q2_qemistree/_collate_fingerprint.py

```diff
@@ -40,7 +40,7 @@ def collate_fingerprint(csi_result: CSIDirFmt, qc_properties: bool = True):
         raise ValueError('Fingerprint file is empty!')
     substructrs = pd.read_table(os.path.join(csi_result, 'fingerprints.csv'),
                                 index_col='relativeIndex', dtype=str)
-    fingerids.index.name = '#featureID'
+    fingerids.index.name = 'id'
```

Comment:
This is `id` here but in the rest of the code it is featureID, selecting either of them should be fine so as long as it is consistent.